stepflow-rs/crates/stepflow-observability/src/lib.rs

```diff
@@ -126,6 +133,14 @@ impl ObservabilityConfig {
             );
         }
 
+        // Validate metrics_enabled requires otlp_endpoint
+        if self.metrics_enabled && self.otlp_endpoint.is_none() {
```

Comment:
Probably worth updating the PR description -- it seems like the dashboard updates only affect our default observability stack -- the main feature is added support for metrics?

Reply:
True - it's only like 2 metrics, but metrics none the less.